Sources/VLCAppDelegate.m

```diff
@@ -249,15 +220,7 @@ - (BOOL)application:(UIApplication *)application
     if([userActivityType isEqualToString:kVLCUserActivityLibraryMode] ||
        [userActivityType isEqualToString:kVLCUserActivityLibrarySelection]) {
 
-        VLCLibraryMode libraryMode = (VLCLibraryMode)[(NSNumber *)dict[@"state"] integerValue];
-
-        if (libraryMode <= VLCLibraryModeAllSeries) {
-            [[VLCSidebarController sharedInstance] selectRowAtIndexPath:[NSIndexPath indexPathForRow:libraryMode inSection:0]
-                                                         scrollPosition:UITableViewScrollPositionTop];
-            [self.libraryViewController setLibraryMode:libraryMode];
-        }
-
-        [self.libraryViewController restoreUserActivityState:userActivity];
+       // [self.libraryViewController restoreUserActivityState:userActivity];
```

Comment:
Please add a comment explaining why this is commented or remove the entire line.